pkg/scorecard_result.go

```diff
@@ -45,12 +46,13 @@ type RepoInfo struct {
 // ScorecardResult struct is returned on a successful Scorecard run.
 // nolint
 type ScorecardResult struct {
-	Repo       RepoInfo
-	Date       time.Time
-	Scorecard  ScorecardInfo
-	Checks     []checker.CheckResult
-	RawResults checker.RawResults
-	Metadata   []string
+	Repo         RepoInfo
+	Date         time.Time
+	Scorecard    ScorecardInfo
+	Checks       []checker.CheckResult
+	RawResults   checker.RawResults
+	ProbeResults []finding.Finding
```

Comment:
Should we stick with `Findings` for consistency?

Reply:
Done.